src/rust/engine/src/nodes.rs

```diff
@@ -1143,16 +1142,30 @@ impl Node for NodeKey {
 
   fn user_facing_name(&self) -> Option<String> {
     match self {
-      NodeKey::Task(ref task) => task.task.display_info.name.as_ref().map(|s| s.to_owned()),
+      NodeKey::Task(ref task) => task.task.display_info.desc.as_ref().map(|s| s.to_owned()),
       NodeKey::Snapshot(_) => Some(format!("{}", self)),
```

Comment:
Could you tweak this one, too, please? It shows up a lot in the video.

Instead of `Snapshot(PathGlobs(globs=('src/python/*.py',)), glob_match_error_behavior=...)`, maybe something like `Resolving glob: 'src/python/*.py'`.

See https://github.com/pantsbuild/pants/pull/9010 for possible inspiration.

Reply:
Hm, this is actually a little bit trickier than I thought it was going to be. This is a good idea but I think I'd rather leave it for a subsequent PR.

Reply:
Sounds good! This PR is a huge improvement either way.